css/properties/offset-path.json

```diff
@@ -167,7 +154,7 @@
                 "version_added": "79"
               },
               "firefox": {
-                "version_added": "71",
+                "version_added": "72",
                 "flags": [
```

Comment:
Should this and any other entry for "layout.css.motion-path.enabled" also be removed? I suspect the data here is just wrong/misleading.

Reply:
This flag actually does seem to be inaccurate, I'll open up another PR to take care of it and explain my findings!